pkg/diagnostics/utils/metrics_utils.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Dapr Authors
+Copyright 2022 The Dapr Authors
```

Comment:
3? Since you're updating this :)

Reply:
OMG my brain didn't register it's 2023 yet.